### PR TITLE
Change sidebar size, more fluid

### PR DESCRIPTION
**Pull request by LukyVj**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -166,6 +167,8 @@ function processHeroProfile(){
       var $this = $(this);
       var datas = $(this).closest('.hit').find('.dump').val(),
           datas = $.parseJSON(datas);
+
+          console.log(datas)
```
Comment (pixelastic) [inline app/app.js]: I think you forgot something here ;)

File: app/app.js [hunk 2]
```diff
@@ -187,24 +190,28 @@ function processHeroProfile(){
       profileDescription.html('<p>'+heroDesc+'</p>');
       profileRealName.html(heroSecretId)
 
+      // Give the profile header the proper images
+      profileHeader.background = 'url(' + heroBackground + ')no-repeat center center / cover';
```
Comment (pixelastic) [inline app/app.js]: Maybe we could put the `background-repeat`, `background-position` definitions in the CSS and only override the `background-image` here? Seems more clean

File: app/app.js [hunk 3]
```diff
@@ -218,7 +225,7 @@ function processHeroProfile(){
       profilePartners.html('');
       var partner = '';
       $.each(heroPartners, function (index, key){
-        partner += "<span class='partner'>"+key+"</span>"; 
+        partner += "<span class='partner'>"+key+"</span>, ";
```
Comment (pixelastic) [inline app/app.js]: I feel like you will have a comma at the end doing that. What you can do is put all your `span` in an array, then do `myArray.join(', ')`.

File: app/app.js (hunk 2 above)
Comment (LukyVj) [inline app/app.js]: That's what I plan too, but for now, I just try to get it as expected, but I agree :+1:

File: app/app.js (hunk 3 above)
Comment (LukyVj) [inline app/app.js]: Ah! clever, will do!

File: app/styles/_profile.scss [hunk 4]
```diff
@@ -1,5 +1,7 @@
 .hero-profile {
   max-width: $profile-bar-width;
+  min-width: calc(#{$profile-bar-width} / 1.6);
```
Comment (pixelastic) [inline app/styles/_profile.scss]: Is `calc` working correctly in browsers now? I mean, without killing all the performance? :older_man:

**Comment by pixelastic** [comment]
Nice.

If you plan on doing a lot of small PRs, try to keep one feature per PR (this one included the sidebar fix as well as some tweaks to the right panel). Or you can just work longer on the design and only submit a bigger PR, that works as well.

**Pull request merged**